source/Tests/Iciclecreek.Bot.Builder.Dialogs.Recognizers.Tests/DynamicListRecognizerTests.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Iciclecreek.Bot.Builder.Dialogs.Recognizers.QLucene;
+using Lucene.Net.Store;
+using Microsoft.Bot.Builder.AI.QnA;
+using Microsoft.Bot.Builder.Dialogs;
+using Microsoft.Bot.Builder.Dialogs.Adaptive;
+using Microsoft.Bot.Builder.Dialogs.Adaptive.Actions;
+using Microsoft.Bot.Builder.Dialogs.Adaptive.Conditions;
+using Microsoft.Bot.Builder.Dialogs.Adaptive.Runtime.Extensions;
+using Microsoft.Bot.Builder.Dialogs.Adaptive.Testing;
+using Microsoft.Bot.Builder.Dialogs.Declarative.Resources;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Iciclecreek.Bot.Builder.Dialogs.Recognizers.Tests
+{
+    [TestClass]
```

Comment:
>TestClass [](http://example.com/codeflow?start=5&length=9)

The primary testing package that the SDK uses is xunit, not the visual studio one.  Could you switch to xunit?